.circleci/config.yml

```diff
@@ -103,7 +103,7 @@ jobs:
   pulumi_up:
     circleci_ip_ranges: true
     docker:
-      - image: cimg/node:18.17
+      - image: cimg/node:20.12
```

Comment:
These match node alpine?
Maybe we should set the nvmrc one to match explicitly?

Reply:
Yeah, alpine always runs its own version which is updated as the upstream image is updated. Minor version of node won't be breaking but yeah we can probably update .nvmrc to match.

Reply:
Yeah in apps I set everything to 20.12 just to be more explicit.
(non blocking though)